addon/plugins/variable-plugin/nodes.ts

```diff
@@ -52,12 +58,40 @@ export const getPNodeExtraAttributes = ({
         (node.attrs[MINIMUM_VALUE_PNODE_KEY] as string) ?? null,
       [MAXIMUM_VALUE_HTML_ATTRIBUTE_KEY]:
         (node.attrs[MAXIMUM_VALUE_PNODE_KEY] as string) ?? null,
+      [WRITTEN_NUMBER_HTML_ATTRIBUTE_KEY]: String(
+        node.attrs[WRITTEN_NUMBER_PNODE_KEY] ?? false
+      ),
     };
   }
 
   return {};
 };
 
+export const contentToDom = ({
+  content,
+  type,
+  node,
+}: {
+  content: string;
+  type: string;
+  node: PNode;
+}) => {
+  if (type === 'number') {
+    if (
+      node.attrs.writtenNumber ||
```

Comment:
as you defined a constant for the name of this attribute (WRITTEN_NUMBER_PNODE_KEY), you should use it here too